bfd/src/test/java/gov/cms/ab2d/bfd/client/R4Test.java

```diff
@@ -0,0 +1,174 @@
+package gov.cms.ab2d.bfd.client;
+
+import ca.uhn.fhir.context.FhirContext;
+import ca.uhn.fhir.parser.IParser;
+import ca.uhn.fhir.rest.api.EncodingEnum;
+import gov.cms.ab2d.fhir.BundleUtils;
+import gov.cms.ab2d.fhir.MetaDataUtils;
+import gov.cms.ab2d.fhir.Versions;
+import org.hl7.fhir.instance.model.api.IBaseBundle;
+import org.hl7.fhir.instance.model.api.IBaseConformance;
+import org.hl7.fhir.r4.model.Bundle;
+import org.hl7.fhir.r4.model.ExplanationOfBenefit;
+import org.hl7.fhir.r4.model.Identifier;
+import org.hl7.fhir.r4.model.Patient;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.TestPropertySource;
+
+import java.time.OffsetDateTime;
+import java.time.ZoneOffset;
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.junit.Assert.fail;
+
+/**
+ * Class used to test against the real R4 BFD server (you need to change values in properties file to get it to work)
+ */
+@SpringBootTest(classes = SpringBootApp.class)
+@TestPropertySource(locations = "/application.bfd.properties")
+public class R4Test {
```

Comment:
Did this class sneak in because none of the tests are enabled and its presence doesn't really make sense.

Reply:
It did.